lib/dom.js

```diff
@@ -945,20 +945,40 @@ declare class HTMLButtonElement extends HTMLElement {
   checkValidity(): boolean;
 }
 
+// http://dev.w3.org/html5/spec-preview/the-textarea-element.html
 declare class HTMLTextAreaElement extends HTMLElement {
+  autofocus: boolean;
   cols: number;
-  defaultValue: string;
+  dirName: string;
   disabled: boolean;
-  form: HTMLFormElement;
+  form: ?HTMLFormElement;
+  maxLength: number;
   name: string;
+  placeholder: string;
   readOnly: boolean;
+  required: boolean;
   rows: number;
+  wrap: string;
+
   type: string;
+  defaultValue: string;
   value: string;
+  textLength: number;
 
+  willValidate: boolean;
+  validity: Object; // Too be more precise, a ValidityState object
```

Comment:
There's a spelling mistake/grammar error here. Comment should read: `To be more precise, a ValidityState object`